jpegexport.inx

```diff
@@ -1,12 +1,13 @@
-<inkscape-extension>
+<?xml version="1.0" encoding="UTF-8"?>
+<inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
     <_name>JPEG Export</_name>
-    <id>id.giac.export.jpg</id>
-    <dependency type="executable" location="extensions">jpegexport.py</dependency>
-    <param name="path" type="string" _gui-text="Export path" _gui-description="Full path to your file, e.g. 'C:\Users\User_Name\Documents\myimage.jpg'"></param>
+    <id>fablabchemnitz.de.jpegexport</id>
```

Comment:
Hi - why did you change id and the submenu where the extension appears? Is this helping users?

Reply:
sorry no. i did it change in my own desktop package because i have a lot of extensions and marked all other extension than default with fablab to have better overview of what i put next to them. for this repo we should keep as is/was

Reply:
:) Thanks!